### ISSUE TITLE
Weapon Cards

### ISSUE THREAD

**Issue by elliottomlinson** [elliottomlinson/rpcg#112]
Another issue for brainstorming. I think we should work on building up some weapon decks so that we can:

- Avoid having to worry about what unarmed ranged combat means
- Balance and compare ranged card attacks against ranged attack actions
- Crank out a bunch of visual-only alternatives for fun

In my mind right now I'm envisioning that these will all be

- #106 

… with a simple title, non-magical photo, and line of rules text describing a passive bonus to ranged or melee damage.

**Comment by elliottomlinson** [comment]
As ***bonus** damage* implies that it's on top of your normal damage - and therefore only dealt on a successful hit - the format of choice for these cards is:

> ✋ Deal `x` bonus melee/ranged damage

where `x` is a value, the result of a roll, or a multiplier, depending on the strength of the weapon.

**Comment by elliottomlinson** [comment]
> where `x` is a value, the result of a roll, or a multiplier, depending on the strength of the weapon.

Extremely powerful weapons where `x` serves as a multiplier could be potentially balanced by clauses in their ✋ effects, i.e.

#### 🔵 Roulette Revolver

> ✋ Roll a d8 every time you make a ranged attack. If you roll an 8, deal triple damage.

**Comment by DuncanUszkay1** [comment]
>Avoid having to worry about what unarmed ranged combat means

I think we should remove that qualifier from rules text to resolve this. That is, all weapon effects apply on all attacks. If you manage to hit them with a thrown rock, your weapon effect applies. If you have no way of dealing damage to an npc, then you cannot use the weapon.

**Comment by DuncanUszkay1** [comment]
I think we should also be cautious to not introduce too many passive heavy cards since they have a greater mental cost. If you have like 6 weapons, each with a verbose effect, combat starts to become a hassle.

**Comment by elliottomlinson** [comment]
Posting this for reference 

> I think if we don't have an active in mind for this we should nix it, or work out this new design pattern in a separate issue.

_Originally posted by @DuncanUszkay1 in https://github.com/elliottomlinson/rpcg/pull/111#r683928582_

**Comment by elliottomlinson** [comment]
> I think we should remove that qualifier from rules text to resolve this

I'm unsure exactly which qualifier you mean, the one in the rules text itself or the "treat symbol as sentence substitution" idea?

**Comment by elliottomlinson** [comment]
My problem is with the "what's the 'active effect' of a gun" question. Intuitively it'd be "shoot it" but in our game that means you can't do that again for the rest of the session. "make shooting it the passive effect" follows - and in both cases I believe you and I would err on the side of 'balancing' it by making it have a powerful and risky active. I'm just wondering if it detracts from the game that not every sword glows when baddies are around - especially when the item's passive is a bonus to a mechanic as basic as movement/damage the player's less likely to cast it anyway.

I think we should have passive-only items in cases like the gun (something small like +2 damage, and then a +4 damage for a 'bigger gun' kind of card). It's a nice way to print and manage the heuristics for our own reference, especially as opposed to combing it with another NPC effect to define their moves. Not only does it give us a nice toolbox for building npcs, it helps simplify the drop system and add variety to the drops for free if we build some NPCs with 'active card' and 'passive card' (like 'attack twice' and '1d4 sword' - which are implicitly balanced so we could just have each npc drop one card at random). The alternative to this being the image-only item cards like we had before - I'd like to collapse the key-item system and this seems like a good way to attach a description and the cardmaster style to plot-relevant (i.e. magic, useful, dangerous, generally *functional*) items (*key gets you into place*, *potion lets you smell further but blows up when you touch grass*, *slap hand +1melee range*, etc.)

Passive cards shouldn't be very strong, maybe good enough that you'd want one on you just in case you run out of cards - in other words I think it's worth having a pool of passive cards to keep the active ones and avoid the finishing-the-session-with-all-your-cards thing.

A relative weakness and abundance of similar passive-only cards suits:
- character creation in terms of backstory, _you have an object, you're better at something_
- attachment to the character and the hand _you don't 'run out' of your signature effect_
- adding breadth to playstyle _especially the more risk-averse_
- helping new players _especially the more risk-averse and those who have played rpgs_
- the early game _stealing a gun from a non-cardmaster wouldn't be that big of a deal later on but would be big 1st session_
- the addition of non-cardmasters as plot-relevant characters _they just have anti-magic socks or a gun or a recipe the townsfolk love or something_
- player confidence _for giving them a remote idea of what to expect from opponents_

I know this is a long post but this is all the stuff I thought we were adding passives for in the first place.

I agree with the simplicity-first philosophy. I've come to like the passive by thinking of it *as* this system. I can't think of a more elegant way to handle things like weapons, armor, blessings, curses, (the equivalents of) racial bonuses and class bonuses, etc. It's the fact we've bothered to codify a passive system at all that makes it warrant promotion from second-class status.

**Comment by DuncanUszkay1** [comment]
I think eventually we will want to have a passive ability system. I'm not sure if that time is now, but 

>t's the fact we've bothered to codify a passive system at all that makes it warrant promotion from second-class status

is a good point towards building it now.

I'll split this into sections for problems we want the new passive cards to address.

### We don't want characters to lose their identity when they lose their cards

Some counterpoints:
- Playing as a normal human in a tabletop can be fun as long as combat isn't overemphasized. Even if players aren't using cards there's still good gameplay to be had in building up to that 'use all of your cards' moment, e.g.
  - Talking to NPCs
  - Exploring the Area
  - Sneaking around
  - Planning out an attack
- The sessions are short, so using all of your cards at once isn't particularly punishing
- The card limitation prevents drawn out combat situations. If you're just a regular Joe without your cards and you run out, you're looking to retreat or enact some kind of last ditch plan to win. I think that's better than a drawn out combat in which players are just using cantrips/relying on passives on their turn. If I can always burn people or if I'm just flat out stronger than them in the long run, then I won't retreat.

If we're going to make these cards to solve this problem, we should be sure that the passives enforce identity. I'm hesitant to add in effects like `+2 damage` since they don't strongly correlate with an identity and have limited meaning in a game that's largely balanced on the fly. `Snapping your fingers produces a spark` for a fire wizard seems like a better fit.

### We're concerned about players who are risk averse not using cards

What kind of gameplay would we create for those people?

### Others 

More to address here that I'll keep thinking about for now 🤔 Particularly how it interacts with physical objects like guns

**Comment by DuncanUszkay1** [comment]
### We want players to know what items do when wielded by NPCs and themselves

What could we accomplish just by adding rules text to some items? I think that having items in your hand just as reminders that you're holding it along with information about object gets us pretty far. 

E.g.
Gun
*Fires bullets that deal 1d6 damage.*

**Comment by elliottomlinson** [comment]
**TL;DR** Adding a heuristic of 'armed melee' (1d6) would solve the underlying issue and save having to introduce any other systems.

The first realization I'm having is that it's not as if we can know all the "game stats" of _items_. If someone picks up a nightstand, which would just be an 'item card' (the kind equivalent to map decor, with just a pic of it on the card) I'd hear their request for a defense bonus against arrows - or the power to block the passive effect of the *Arsonist's Brass Knuckles* at the cost of the item, etc. This makes me worry less about them needing info on them since I'd rather not enter into a world where we're printing 🛡+2 ♥+1 👊+1 🏹+2 ⚔+1 and all that on item cards - especially since it works so well with the board, if we can leave them as just an image then we should.

This brought me to back to the 'gun' example. In the case of the gun, we'd know that it was only the means to deal the default amount of ranged damage. Most guns would be about the same anyway - if they were really so different we could just give them an active and a passive and then they're just a normal card. I also realized that I was short-sighted in my appraisal of our approach to cast effects, in having the idea that the active of a sniper rifle could just be something like 'guarantee a hit on your next turn' i.e. it doesn't have to be 'high damage single move'.

This line of thinking has helped me recognize what my major concern was in the first place - *you should deal more melee damage when armed than unarmed*. I was thinking about *sledgehammer* and *sword* and that sort of thing when I started worrying about passive-only. If we added an 'armed melee' heuristic (1d6, to beat the unarmed 1d4 and to match the ranged damage one) we could have melee weapons just be pictures on item cards, while still offering an advantage to players. 

We could compact the system by saying 'unarmed' = 1d4, 'armed' = 1d6. Unarmed ranged damage has a clearer meaning than I thought - just 'without a dedicated thing to cause it'. This way a thrown book doesn't deal bullet damage.

**Comment by elliottomlinson** [comment]
To address my original list of concerns:

> - character creation in terms of backstory, _you have an object, or you're better at something_

This point isn't really in favour of passive-only. It would be disappointing to have a character start with an object in place of a third of their potential skill. Just have them start green-heavy and they'll be given identity cards through that.

> - attachment to the character and the hand _you don't 'run out' of your signature effect - if you're the wizard who sets shit on fire with the passive on the card that he could cast to fireball he's only going to do that when he's up against death or he'll cost himself his brand_

This is and should be the player's choice

> - adding breadth to playstyle _especially the more risk-averse_
> - helping new players _especially the more risk-averse and those who have played rpgs_

Solved by adding more passives/green cards

> - the early game _stealing a gun from a non-cardmaster wouldn't be that big of a deal later on but would be big 1st session_

It already is, as dealing ranged damage is tough without a weapon

> - the addition of non-cardmasters as plot-relevant characters _they just have anti-magic socks or a gun or a recipe the townsfolk love or something_

In this case the anti-magic socks are clearly a magic item (maybe that the NPC doesn't know how to cast, would be an easy way to give it an active and explain it away) and the recipe is just an item that happens to be plot relevant. Items could have some trivial info in their description box too, I'll make sure our importer can handle that when I re-do it.

> - player confidence _for giving them a remote idea of what to expect from opponents_

If we want the card to be identity-building above all else, we can just give it a slightly weaker Cast - but I also suspect this will be something naturally emergent anyway as we aren't that fantastic at predicting the approaches players will take.
